README.md

```diff
@@ -1,2 +1,82 @@
 # queries
 Hosts reusable log queries which are built into a single queries.json file.
+
+
+# Query database structure
+
+```yaml
+queries:
+  query1:  # that is query id (sorted by id)
+    name: foo  # required
+    regex: bar  # optional
+    pattern: aaa # optional (either regex or pattern required)
+    url: ... # str or list[str], issue links to lp, bz,... (optional)
+    category: str  # required, one of ...
```

Comment:
what does mean "category" here? "str" - string? for what? please add more details, nothing is clear

Reply:
I seen category as the section that would be used when creating the html report.  Imagine that we have 10 matches on a single build and sorting them by their id is not ideal because we likely want to sort them by category first and by name as second criteria.

This means that we could list "infra" issues in a single group, one for "code", maybe one like "overcloud"? 

Opposed to tag, category cannot have multiple values. 

If you find the term confusing, I do have two other alternatives in mind "topic" or either "section".

Either way, we would want to enforce which strings are accepted there as we do not want it to be free-form, as too many sections would negate the benefit of grouping inside reports. I hope to never reach more than ten values there. If we need to rename them it would be easy as all of them are defined only inside this repo.

Reply:
We should avoid multiple matches, it doesn't make our job easier. Duplicate matches should be squashed.